### PR TITLE
updated scripts to use curl and identify the OS and architecture from the machine

### PR DESCRIPTION
**Pull request by KirbyKatcher**
## Why?

this changes is needed in order for the openapi2beans build scripts can run on machines other than darwin arm64 machines, so that it can be run by anyone

### CODE REVIEW

**Comment by galasa-team** [comment]
Build failed, see http://localhost:8001/api/v1/namespaces/tekton-pipelines/services/tekton-dashboard:http/proxy/#/namespaces/galasa-build/pipelineruns/repo-buildutils-pr-44-2qzfv for details. If you are unable to do so, please contact a member of the Galasa team.

File: openapi2beans/JavaChecker/build-locally.sh [hunk 1]
```diff
@@ -43,11 +43,37 @@ warn() { printf "${tan}➜ %s${reset}\n" "$@" ;}
...
+            error "Unsupported operating system is in use. $raw_os"
+            exit 1
+    esac
+
+    architecture=$(uname -m)
+    case $architecture in
+        aarch64)
+            architecture="arm64"
+            ;;
+        x86_64)
```
Comment (Wyvinar) [inline openapi2beans/JavaChecker/build-locally.sh:66]: Other way around here I believe, we're going with x86_64 instead

**Comment by galasa-team** [comment]
Build successful

File: openapi2beans/JavaChecker/build-locally.sh (hunk 1 above)
Comment (KirbyKatcher) [reply openapi2beans/JavaChecker/build-locally.sh]: done

**Pull request merged**